Maquetado/scss/styles.scss

```diff
@@ -0,0 +1,173 @@
+body {
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+* {
+  font-family: 'Roboto', sans-serif;
+}
+
+.main-title {
+  font-size: 24px;
```

Comment:
You should have a `_sizes.scss` file, the exact same you have with colors, and define all the sizes that could be variable, like the font-size for titles, subtitles, etc